### ISSUE TITLE
DescribableObject tests break subsequent unit tests

### ISSUE THREAD

**Issue by Zee2** [microsoft/MixedRealityToolkit-Unity#11119]
## Describe the bug

When running unit tests in series, `DescribableObjectTests` dbreak subsequent unit tests that are present in the same run. This is because `DescribableObjectTests` doesn't subclass `RuntimeInputTest` and something suspicious happens related to the rig/playspace/XROrigin setup.

## To reproduce

Run unit tests. Subsequent tests after the `DescribableObjectTests` fail.

## Expected behavior

Tests should not fail.

## Screenshots

![image](https://user-images.githubusercontent.com/5544935/196861471-223a6d64-365e-43b0-abd4-5550f9520d21.png)

## Your setup

MRTK3 latest

## Target platform

Test harness
